packages/ember-toucan-core/src/components/form/fields/input.ts

```diff
@@ -47,10 +52,29 @@ interface ToucanFormInputFieldComponentSignature {
     default: [];
     label: [];
     hint: [];
+    secondaryInformation: [
+      {
+        CharacterCount: WithBoundArgs<typeof CharacterCount, 'current'>;
+      }
+    ];
   };
 }
 
 export default class ToucanFormInputFieldComponent extends Component<ToucanFormInputFieldComponentSignature> {
+  @tracked count = 0;
```

Comment:
```suggestion
  @tracked count = this.args.value?.length ?? 0;
```

I think (untested) with this you wound't need to override the constructor below, so would make it a bit simpler. A class filed initializer is basically syntactic sugar for setting the class field at the bottom of the constructor, as you do in L75, so you can do either one or the other, don't need to do both (I am not even 100% if the initializer above setting count to zero is not overriding what you do in L75!? 🤔 ), and an initializer is usually more terse and easier to read IMHO.